cluster-up/cluster/kind-k8s-1.17.0-ipv6/provider.sh

```diff
@@ -11,4 +13,26 @@ source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
 function up() {
     cp $KIND_MANIFESTS_DIR/kind-ipv6.yaml ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml
     kind_up
+    # remove the rancher.io kind default storageClass
+    _kubectl delete sc standard
+
+    nodes=$(_kubectl get nodes -o=custom-columns=:.metadata.name | awk NF)
+    for node in $nodes; do
+        # Create local-volume directories, which, on other providers, are pre-provisioned.
+        # For more info, check https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/STORAGE.md
+        for i in {1..10}; do
+            node=$node mount_disk $i
+        done
+        $DOCKER exec -it $node bash -c "chmod -R 777 /var/local/kubevirt-storage/local-volume"
+    done
+
+    # create the `local` storage class - which functional tests assume to exist
+    _kubectl apply -f $KIND_MANIFESTS_DIR/local-volume.yaml
+}
+
+function mount_disk() {
+    local idx=$1
+    $DOCKER exec -it $node bash -c "mkdir -p /var/local/kubevirt-storage/local-volume/disk${idx}"
```

Comment:
Same for node
local node=$2 so it's easier to follow.

Reply:
Done.